### ISSUE TITLE
Can't find lutris install on Debian Trixie

### ISSUE THREAD

**Issue by tohaz** [satisfactorymodding/SatisfactoryModManager#247]
lutris is installed via apt. User can start it via command line "lutris". Lutris has EGS Satisfactory installation which runs fine. Except Mod Manager fails to see it and prints "ficsitcli/metadata.go:27 failed to find installations error="failed to run lutris -lj: exit status 1"  on startup

[SMMDebug-2024-10-14-12-50-53.zip](https://github.com/user-attachments/files/17364351/SMMDebug-2024-10-14-12-50-53.zip)

binary on my system is at /usr/games/lutris

**Comment by mircearoata** [comment]
This is caused by a combination of issues that we've debugged [over on discord](https://discord.com/channels/555424930502541343/1294659585797259287) for the past couple of days, and just got it to work.

The current release doesn't pass the environment variables to the lutris process, causing it not initialize GTK properly because it is missing the `DISPLAY` env var.
But passing all env vars failed when using the AppImage, because the AppImage prepends its own libraries path to `LD_LIBRARY_PATH` as well as setting `PYTHONHOME` to point inside the AppImage, even though it does not contain any python libraries.
After handling that by removing AppImage entries from the two env vars when calling lutris, we found that the Epic install handler was also not correctly adding the wine prefix to the Windows paths referenced by Epic.

A fixed build is available in the latest commit's Github Actions artifacts, https://github.com/satisfactorymodding/SatisfactoryModManager/actions/runs/11329649015/artifacts/2053658702
